recipes/relion/build.sh

```diff
@@ -0,0 +1,9 @@
+CMAKE_ARGS+=" -DCMAKE_INSTALL_PREFIX=${PREFIX}"
+CMAKE_ARGS+=" -DCMAKE_BUILD_TYPE=Release"
+CMAKE_ARGS+=" -DCMAKE_CUDA_HOST_COMPILER=${CXX}"
+CMAKE_ARGS+=" -DFORCE_OWN_FLTK=OFF"
+CMAKE_ARGS+=" -DFORCE_OWN_FFTW=OFF"
+CMAKE_ARGS+=" -DBUILD_SHARED_LIBS=OFF"
+
+cmake $CMAKE_ARGS $SRC_DIR
+make -j $CPU_COUNT
```

Comment:
I'd write it like this:

```suggestion
CUSTOM_CMAKE_ARGS="-DFORCE_OWN_FLTK=OFF"
CUSTOM_CMAKE_ARGS+=" -DFORCE_OWN_FFTW=OFF"
CUSTOM_CMAKE_ARGS+=" -DBUILD_SHARED_LIBS=OFF"

cmake ${CMAKE_ARGS} ${CUSTOM_CMAKE_ARGS} "$SRC_DIR"
make -j $CPU_COUNT
```